lib/ngx/errlog.md

```diff
@@ -210,6 +211,28 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filter_level
+--------------------
+**syntax:** *log_level = log_module.get_sys_filter_level()*
+
+**context:** *any*
+
+Return the system's default filter level as an integer. Later, the returned
+value could be used as an Nginx log level constant. For example:
```

Comment:
Maybe we could keep `Nginx` here, because the `set_filter_level` already used `Nginx log level constants` in its document. Just follow existing style.